raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -124,6 +125,13 @@ contract TokenNetwork is Utils {
         uint256 locked_amount;
     }
 
+    struct ParticipantSettleInput {
```

Comment:
imho we could go with a shorter name here:
`SettleInput` (having a mandatory `participant` argument) should be completely self-explanatory :)